charts/admission-controller/templates/webhook/certificate.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.webhook.ssl.certManager.enabled }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  labels:
+    {{- include "admissionController.webhook.labels" . | nindent 4 }}
+  name: {{ include "admissionController.webhook.fullname" . }}
+  namespace: {{ include "admissionController.namespace" . }}
+spec:
+  dnsNames:
+  - {{ include "admissionController.webhook.fullname" . }}
+  - {{ include "admissionController.webhook.fullname" . }}.{{ include "admissionController.namespace" . }}.svc
+  issuerRef:
+    {{- with .Values.webhook.ssl.certManager.issuer.group }}
```

Comment:
I've deliberated done it this we so we can easily add a boolean parameter like `webhook.ssl.certManager.issuer.create` later to determine whether we should create an issuer.